packages/documentation-spa/blog/2021-02-08-state-of-one-platform-jan-2021.md

```diff
@@ -0,0 +1,173 @@
+   </td>
+   <td>Not Started
+   </td>
+  </tr>
+</table>
+
+### Key highlights
+The team was able to close 11 JIRAs across a range of development priorities and was able to push 13 issues to “Development Complete”. The highlights are :
+
+- **Application development and readiness**
+  - **Native Apps**:
+    - **API Gateway**: API update to manage API keys generated.
+    - **Feedback**: Next version of Feedback app developed, Dynamic target integration with Gitlab, JIRA, GitHub ready to use.
+    - **Developer Console**: New microservice implementation in progress which aggregates all the SPA and microservices configurations at a single place.
+    - **Notifications**: Enhancements in SPA & Microservice are completed. Implementation underway for requirements of DXP Drupal Properties.
+    - **User Groups and Authorization**: Enhancements implemented.
+    - **Home**: Categorization of deployed SPAs into Native & Non-native categories.
+  - **Non Native Apps**:
+    - **Research Repository**: Included additional content type in the results  & Enhancements in analytics dashboard.
+    - **DSAL**: Released to production on One Platform. Ongoing development for DSAL Dashboard.
+    - **MoD Handovers**: Feedback addressed which are reported via feedback module.
+  - **Component Library**:
+    - **Storybook**: Basic support of storybook for the components.
+- **Onboarding**:
+  - Onboarding app list readiness
+  - GDPR deployed on Stage
+  - In conversation with Pantheon and IR Dashboard
+
+### What's coming?
+
+- **SPA Onboarding**
+  - **GDPR**: Deploy GDPR on the Production environment.
+  - **Pantheon**: Discussions underway. Timelines will be communicated in the next Jan’21 blog.
+  - **Life Cycle**: Initiate the conversation with  Feng Wang (fwang@redhat.com).
+
+- **Development and Adoption Priorities**
+  - **DSAL**: Stats and Ansible Dashboard development and release to users(continued). Completion of DSAL application.
+  - **Feedback**: Integration with Developer Console and Explore opportunities to use Qualtrics.
+  - **Notifications (impact on OKR)**: Integration with Developer Console for consistent developers experience.
+  - **Developer Console (impact on OKR)**: Implementation of Developer Console SPA based upon approved UX designs. Feedback/Notifications top priority(continued)
+  - **Research Repository**: Further enhancements and feature development. Completion of Research Repository application.
+  - **Status Dashboard**: A simple status page for the One Platform properties (SPA/Microservices) uptime and performance.
+
+- **One Platform Test Automation**: Automated end-to-end testing for One Platform initial set of modules, in collaboration with the QE team. Create one POD in MPaaS to deploy a test runner app created by the QA team to test infrastructure readiness.
+
+For more updates related to One Platform, please check out the One Platform Blog at: [one.redhat.com/blog](https://one.redhat.com/get-started/blog)
```

Comment:
Also, I think we need at least a redirect from one.redhat.com/blog to /get-started/blog.
```suggestion
For more updates related to One Platform, please check out the One Platform Blog at: [one.redhat.com/get-started/blog](https://one.redhat.com/get-started/blog/)
```

Reply:
Ok. I will create an issue for the redirection. As of now suggested changes are pushed.